indigo_api/exporters.py

```diff
@@ -172,10 +174,13 @@ def render(self, document, element=None):
             # fix tables to have the right number of columns and rows; else FOP complains
             self.resize_tables(document.doc)
             # write the XML to file
-            xml = etree.tostring(document.doc.main, encoding='unicode')
+            xml_string = etree.tostring(document.doc.main, encoding='unicode')
+            xml_bytes = xml_string.encode('utf-8')
             xml_file = os.path.join(tmpdir, 'raw.xml')
-            with open(xml_file, "wb") as f:
-                f.write(xml.encode('utf-8'))
+            # use io.open, BytesIO, getbuffer to avoid XML parse error in production
+            with io.open(xml_file, "wb") as f:
```

Comment:
can you link to the issue this is trying to solve? I don't see how this change will fix anything.

Reply:
Ah https://github.com/laws-africa/indigo-lawsafrica/issues/1012